Toolchain/CMake/GNUToolchain.txt.in

```diff
@@ -0,0 +1,41 @@
+list(APPEND CMAKE_MODULE_PATH "@SERENITY_SOURCE_DIR@/Toolchain/CMake")
+
+set(CMAKE_SYSTEM_NAME SerenityOS)
+set(CMAKE_SYSTEM_PROCESSOR "@SERENITY_ARCH@")
+set(CMAKE_CROSS_COMPILING 1)
```

Comment:
This is UB. Please see the documentation of [CMAKE_CROSSCOMPILING](https://cmake.org/cmake/help/latest/variable/CMAKE_CROSSCOMPILING.html).

Reply:
removed in ea349c1e54a2d17a84219347401eef99cf050790